src/bin/workflow.ts

```diff
@@ -25,22 +43,48 @@ export function coerceUserCreatePullRequestOptions(): CreatePullRequestUserOptio
   };
 }
 
+/**
+ * map yargs to user pull request otions
+ */
+export function coerceUserCreateReviewRequestOptions(): CreateReviewCommentUserOptions {
+  return {
+    repo: yargs.argv.upstreamRepo as string,
+    owner: yargs.argv.upstreamOwner as string,
+    pullNumber: yargs.argv.pullNumber as number,
+  };
+}
+
+async function createCommand() {
+  const options = coerceUserCreatePullRequestOptions();
+  const changes = await git.getChanges(yargs.argv['git-dir'] as string);
+  const octokit = new Octokit({auth: process.env.ACCESS_TOKEN});
+  await createPullRequest(octokit, changes, options, logger);
+}
+
+async function reviewCommand() {
+  const reviewOptions = coerceUserCreateReviewRequestOptions();
+  const diffContents = await git.getDiffContents(
+    yargs.argv['git-dir'] as string
+  );
+  const octokit = new Octokit({auth: process.env.ACCESS_TOKEN});
+  await reviewPullRequest(octokit, diffContents, reviewOptions, logger);
+}
+
 /**
  * main workflow entrance
  */
 export async function main() {
   try {
     setupLogger();
-    const options = coerceUserCreatePullRequestOptions();
     if (!process.env.ACCESS_TOKEN) {
       throw Error('The ACCESS_TOKEN should not be undefined');
     }
-    const octokit = new Octokit({auth: process.env.ACCESS_TOKEN});
-    let changes: Changes;
     switch (yargs.argv._[0]) {
       case CREATE_PR_COMMAND:
-        changes = await git.getChanges(yargs.argv['git-dir'] as string);
-        await createPullRequest(octokit, changes, options, logger);
+        createCommand();
+        break;
+      case REVIEW_PR_COMMAND:
```

Comment:
You can get pretty fancy and provide a handler to the command itself, rather than using a switch like this:


```js
.command('command', 'description of command', (yargs) => {
  yargs.option('foo', {
    type: 'string'
   })
}, (argv) => {
  // this code runs when the command is invoked.
})
````

Here's an example of a really fancy command line application built with yargs, that encapsulates commands in their own files:

https://github.com/redwoodjs/redwood/tree/main/packages/cli/src/commands